Python/ds/Binary-Search-Tree-Validation.py

```diff
@@ -0,0 +1,92 @@
+import typing
```

Comment:
Add a beginner pitch about the Algorithm that you are implementing

Reply:
Sure.